Update on the FAIR principles (#2849)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/rdm-fair.md

(rr-rdm-fair-community)=
## Community involvement

Various online resources are provided for people who are working in the life sciences, to guide them in ensuring FAIRness in their data, providing them with tools and advice for good data management at various stages of their work. Two prominent ones include: 

> * Under the [FAIR Cookbook](https://faircookbook.elixir-europe.org/content/home.html), several resources are provided by a thriving community of professionals and academics working in the life sciences, in offering guidance and assistance in FAIR data management for anyone working in the field. It is designed to serve a variety of audience types and involved in different stages of data management life cycle.

Reviewer: ```suggestion
* Under the [FAIR Cookbook](https://faircookbook.elixir-europe.org/content/home.html), several resources are offering guidance and assistance in FAIR data management.
 The FAIR Cookbook is designed to serve a variety of audience types and involved in different stages of data management life cycle.
The FAIR Cookbook is developed and maintained by life sciences professionals, both in the academia and industry sectors, including members of the ELIXIR community. 
```

> * Under [ELIXIR Research Data Management Kit (RDMkit)](https://rdmkit.elixir-europe.org/), resources are provided for life scientists to guide them in better management of their research data in adhering to the FAIR Principles. It is an attempt to help researchers work at different capacities, both in individual and collaborative workspaces.

Reviewer: ```suggestion
* Under [ELIXIR Research Data Management Kit (RDMkit)](https://rdmkit.elixir-europe.org/), resources are provided for life scientists to guide them in better management of their research data in adhering to the FAIR Principles. 
It is an attempt to help researchers work at different capacities, both in individual and collaborative workspaces.
The RDMkit is open for suggestions from anyone, as long as they abide by the [contributor responsibilities](https://rdmkit.elixir-europe.org/how_to_contribute).
```

Since both are community-driven resources, they welcome contributions from the thriving Life Sciences community to enrich their guidebooks. While [FAIR Cookbook](https://faircookbook.elixir-europe.org/content/recipes/community.html) is developed and maintained by life sciences professionals, both in the academia and the industry sectors, including members of the ELIXIR community, [RDMkit](https://rdmkit.elixir-europe.org/how_to_contribute) is open for suggestions from anyone within the community and outside through various ways, as long as they abide by the contributor responsibilities.

> Although started by a community operating in the life science, the FAIR principles have rapidly been adopted by publishers, funders, and pan-disciplinary infrastructure programmes and societies.

Reviewer: The citation from the top of this page could be used for this - but we're discussing a bit on Slack and we think the sentence could actually go, so no citation needed ;)

> [FAIR Cookbook](https://faircookbook.elixir-europe.org/content/recipes/community.html) is developed and maintained by life sciences professionals, both in the academia and industry sectors, including members of the ELIXIR community. [RDMkit](https://rdmkit.elixir-europe.org/how_to_contribute) is open for suggestions from anyone from within the community and outside of it, as long as they abide by the [contributor responsibilities](https://rdmkit.elixir-europe.org/how_to_contribute).

Reviewer: ```suggestion

```

Many groups and organisation are working to define guidance and tools to help researchers and other stakeholders (like librarians, funders, publishers, and trainers) make data more FAIR.

> If you are interested in participating in these communities there are two global initiatives that act as umbrella organizations and reference points for many discipline-specific efforts: [GOFAIR](https://www.go-fair.org) and the [Research Data Alliance (RDA)](https://www.rd-alliance.org).

Reviewer 2: ```suggestion
If you are interested in participating in these communities, there are two global initiatives that act as umbrella organisations and reference points for many discipline-specific efforts, including the ones listed above: [GOFAIR](https://www.go-fair.org) and the [Research Data Alliance (RDA)](https://www.rd-alliance.org).
```

> Although initiated by open science communities operating in the life sciences, the FAIR principles have rapidly been adopted by publishers, funders, and pan-disciplinary infrastructure programmes and societies to maximise the openness and reusability of the data.

Reviewer: ```suggestion

```

> If you are interested in participating in these communities, there are two global initiatives that act as umbrella organisations and reference points for many discipline-specific efforts, including the ones listed above: [GOFAIR](https://www.go-fair.org) and the [Research Data Alliance (RDA)](https://www.rd-alliance.org).

Reviewer 2: ```suggestion
There are two global initiatives that act as umbrella organisations and reference points for many discipline-specific efforts, including the ones listed above: [GOFAIR](https://www.go-fair.org) and the [Research Data Alliance (RDA)](https://www.rd-alliance.org).
```

* Under GOFAIR, there are many [Implementation Networks (INs)](https://www.go-fair.org/implementation-networks) committed to implementing the FAIR principles.

> * Under the RDA, there are several groups tackling different aspects relevant to the RDM life cycle. Among these, one [group](https://www.rd-alliance.org/groups/fair-data-maturity-model-wg) is reviewing existing efforts, building on them to define a standard set of common assessment criteria for the evaluation of FAIRness.

Reviewer 2: ```suggestion
* Under the RDA, there are several groups tackling different aspects relevant to the RDM life cycle. Among these, one group, the [FAIR Data Maturity Model Working Group](https://www.rd-alliance.org/groups/fair-data-maturity-model-wg) is reviewing existing efforts, building on them to define a standard set of common assessment criteria for the evaluation of FAIRness.
```
